.vscode/tasks.json

```diff
@@ -1,30 +1,36 @@
 {
```

Comment:
I believe that this changes are from master (file has been remove from the latest). I can revert them if need be.